index.html

```diff
@@ -24,83 +26,209 @@ <h3 class="name1">AB</h3>
+                    <div class="name2">
+                        <h1>Abigiya Tadesse</h1>
+                    </div>
+                    <div class="occupation">
+                        <h2>Software Developer</h2>
+                    </div>
+                    <div class="description">
+                        <p>
+                            I can help you build a product , feature or website Look through of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+                        </p>
+                    </div>
+                </div>
             </div>
-            <div class="personal-description">
-                <div class="name2">
-                    <h1>Abigiya Tadesse</h1>
+        </div>
+    </section>
+    <section>
+        <div class="line"></div>
+        <div class="skills-list">
+            <div class="lang-skill">
+                <div class="language">
+                    <h3>Language</h3>
+                    <ul>
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>Html</li>
+                        <li>Css</li>
+                    </ul>
                 </div>
-                <div class="occupation">
-                    <h2>Software Developer</h2>
+                <div class="skills">
+                    <h3>Skills</h3>
+                    <ul>
+                        <li>Codekit</li>
+                        <li>Github</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
                 </div>
-                <div class="description">
-                    <p>
-                        I can help you build a product , feature or website Look through of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
-                    </p>
+            </div>
+            <div class="ecl-frame">
+                <div class="frameworks">
+                    <h3>Frameworks</h3>
+                    <ul>
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul>
                 </div>
+                <div class="ellipse2"></div>
             </div>
         </div>
     </section>
-    <section class="skills-list">
-        <div class="lang-skill">
-            <div class="language">
-                <h3>Language</h3>
-                <ul>
-                    <li>Javascript</li>
-                    <li>Ruby</li>
-                    <li>Html</li>
-                    <li>Css</li>
-                </ul>
+    <section>
+        <div class="work">
+            <h1 class="work-title">MY RECENT WORK</h1>
```

Comment:
- [ ] Kindly consider having a single h1 in a webpage. h1 is used to indicate the most important heading (for this case, your name), having more than one may lead to spamming. kindly change this to one of the least important headings.